statistics/most_visited_countries.rb

```diff
@@ -20,10 +20,10 @@ def query
         WHERE competition.countryId -- Ignore Multiple Countries used for continental FMC competitions.
           NOT IN ('XA', 'XE', 'XF', 'XM', 'XN', 'XO', 'XS', 'XW')
         GROUP BY personId
-        ORDER BY visited_countries DESC
         LIMIT 100
```

Comment:
We need to order here, because otherwise `LIMIT 100` will take 100 random entries, rather than the top ones. It's fine to have the order by in both places.

Reply:
Good catch, thanks! 

Since I can't load any mysql2 gem after 0.5.4 I've been unable to test it myself, sorry. I've looked for help in the usual places, but so far no luck, 0.5.5 refuses to load :(

Reply:
Curious, I updated to 0.5.5 because 0.5.4 failed to build on macOS.

Reply:
Hahaha, that's amazing, since I'm on macOS myself! 

I'm on the latest Monterey, v12.6.7. I've been meaning to update to Ventura now that it's matured a bit, but I never seem to get round to it. What version of macOS are you running?

Reply:
I'm on Ventura v13.4.1 :)

Reply:
OK, then I guess I'll have to bite the bullet and do the upgrade. I was already thinking I might just nuke ruby and mysql and install them from scratch, so I'll get that for free 🤪